### PR TITLE
Add og image to Web components storybook

### LINKED ISSUE #5427: Update Storybook social media preview image
## Task
Mike asks to have the social sharing image of storybook updated. 
Currently it's still showing the Carbon v9 cover: 

Carbon's location. So ours will be something similar? Would need advice.
`https://github.com/carbon-design-system/carbon/blob/main/packages/react/.storybook/manager-head.html`

### PR DESCRIPTION
**Pull request by shixiedesign**
### Related Ticket(s)

Closes #5427

### Description

Added a little image for open graph meta tag so that when storybook link is shared on social media / slack / anywhere, preview image of link will be updated to the new IDL: 

Artboard 23

### CODE REVIEW

**Comment by shixiedesign** [comment]
It works!! 

image

File: packages/web-components/.storybook/manager-head.html [hunk 1]
```diff
@@ -14,7 +14,10 @@
 <meta property="og:title" content="Carbon for IBM.com Web Components">
 <meta property="og:site_name" content="Carbon for IBM.com Web Components">
 <meta property="og:description" content="This is the Web Components (custom elements) implementation of Carbon for IBM.com.">
-
+<meta
+  property="og:image"
+  content="https://media.github.ibm.com/user/52210/files/52a0cb80-8256-11eb-9bda-0e7a95354168"
```
Comment (jeffchew) [inline packages/web-components/.storybook/manager-head.html:19]: Let me know if you want me to upload to a place a bit more permanent (e.g. akamai server)

Comment (shixiedesign) [reply packages/web-components/.storybook/manager-head.html]: Oo yes that would be nice! How do i do that lol

Comment (shixiedesign) [reply packages/web-components/.storybook/manager-head.html]: James said there's a place for all placeholder images. I'd also be interested in updating those in the future. So Might be worth for me to figure it out

**Comment by shixiedesign** [comment]
Tested and works! 
image

File: packages/web-components/.storybook/manager-head.html [hunk 2]
```diff
@@ -14,7 +14,14 @@
...
 <meta property="og:site_name" content="Carbon for IBM.com Web Components">
 <meta property="og:description" content="This is the Web Components (custom elements) implementation of Carbon for IBM.com.">
-
+<meta
+  property="og:image"
+<<<<<<< HEAD
+  content="https://1.www.s81c.com/common/carbon-for-ibm-dotcom/i/carbon-for-ibmcom-wc.png"
+=======
+  content="https://user-images.githubusercontent.com/15144993/110886019-ea81d200-82b5-11eb-973e-2b3d60d5e60d.png"
+>>>>>>> 68840ef3d8870f999f08d43aa1e9d9535573e8b8
```
Comment (asudoh) [inline packages/web-components/.storybook/manager-head.html:23]:
```suggestion
  content="https://1.www.s81c.com/common/carbon-for-ibm-dotcom/i/carbon-for-ibmcom-wc.png"
```

File: packages/web-components/src/components/card/__stories__/README.stories.mdx [hunk 3]
```diff
@@ -26,8 +26,13 @@ import '@carbon/ibmdotcom-web-components/es/components/card/card-heading.js';
 
 ```html
 <dds-card href="https://example.com">
+<<<<<<< HEAD
+  <span slot="eyebrow">Eyebrow text</span>
+  <span slot="heading">Lorem ipsum dolor sit amet</span>
+=======
   <dds-card-eyebrow>Eyebrow text</dds-card-eyebrow>
   <dds-card-heading>Lorem ipsum dolor sit amet</dds-card-heading>
+>>>>>>> 5890aaf77d613555f9b1a0c36c0f4abbc97236cf
```
Comment (asudoh) [inline packages/web-components/src/components/card/__stories__/README.stories.mdx:35]:
```suggestion
  <dds-card-eyebrow>Eyebrow text</dds-card-eyebrow>
  <dds-card-heading>Lorem ipsum dolor sit amet</dds-card-heading>
```

**Review by asudoh — COMMENTED** [review]
Thank you for doing this @shixiedesign!

**Comment by shixiedesign** [comment]
Thank you @asudoh for the help!!!

**Review by asudoh — APPROVED** [review]
Thanks @shixiedesign!

**Pull request merged**